src/core/message.ts

```diff
@@ -23,22 +23,34 @@ export type MessageHeaders = Headers;
 export type MessageMetadata = Metadata;
 
 /**
- * {@link OperationMetadata} type for a chat messages {@link Operation}.
+ * {@link OperationMetadata} type for a chat message. Contains information about an update or deletion operation.
  */
 export type MessageOperationMetadata = OperationMetadata;
 
 /**
  * Represents the detail of a message deletion or update.
  */
-export interface Operation {
+export interface Version {
```

Comment:
We've named it in `MessageVersion` in the spec, so should probably do the same here